wheelhouse.txt

```diff
@@ -8,6 +8,8 @@ pip<22.1;python_version >= '3.8'
 # for trusty
 Jinja2==2.10;python_version >= '3.0' and python_version <= '3.4' # py3 trusty
 Jinja2==2.11;python_version == '2.7' or python_version == '3.5'  # py27, py35
+# Focal failures - https://github.com/juju-solutions/layer-basic/issues/210
+Jinja2==2.11;python_version == '3.8'                             # py38 focal
```

Comment:
This doesn't look quite right.  In order not to end up with multiple requirements lines, the guards need to be fully exclusive.  I think this and the next line should be:

```
Jinja2==2.11;python_version >= '3.6 and <=  '3.8'
Jinja2; python_version > '3.8'

```

Reply:
Thanks for your input, fixed.

Reply:
```
Jinja2==2.11;python_version >= '3.6' and python_version <= '3.8' # py35, focal
Jinja2;python_version > '3.8'
```

As the example above produces errors, '3.6' unterminated string, and needed **python_version** <= '3.8'